components/SiteNavbar.vue

```diff
@@ -1,38 +1,231 @@
 <template>
-  <nav class="flex flex-wrap flex-row-reverse justify-between border-b border-gray-300 px-5 py-3 lg:flex-row">
-    <div class="flex lg:hidden" @click="mobileNavIsActive = !mobileNavIsActive">
-      <button class="flex items-center border border-gray-300 rounded px-4 py-1 text-gray-600 hover:text-black hover:border-black hover:bg-gray-100">
-        <svg class="fill-current h-3 w-3" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
-          <title>Menu</title>
-          <path d="M0 3h20v2H0V3zm0 6h20v2H0V9zm0 6h20v2H0v-2z"/>
+  <nav class="border-b border-gray-300">
+    <div
+      class="
+        mx-auto
+        container
+        flex flex-wrap flex-row-reverse
+        justify-between
+        px-5
+        py-3
+        lg:flex-row
+      "
+    >
+      <div class="flex lg:hidden">
+        <a
+          class="
+            rounded
+            px-3
+            py-2
+            m-1
+            text-gray-600
+            lg:my-0
+            hover:text-gray-800 hover:bg-gray-100
+          "
+          href="#"
+          >Log In</a
+        >
+        <a
+          class="
+            bg-gray-600
+            rounded
+            px-3
+            py-2
+            m-1
+            text-gray-100
+            lg:my-0
+            hover:text-white hover:bg-gray-900
+          "
+          href="#"
+          >Sign Up</a
+        >
+        <button
+          class="
+            flex
+            items-center
+            border border-gray-300
+            rounded
+            px-4
+            py-1
+            text-gray-600
+            hover:text-black hover:border-black hover:bg-gray-100
+          "
+          @click="mobileNavIsActive = !mobileNavIsActive"
+        >
```

Comment:
could the padding be changed from `px-4 py-1` to `px-3 py-2` to make this button more consistent with the others?